### PR TITLE
Custom select

### PR DESCRIPTION
**Pull request by Lansyfonseka**
Create custom Select component. 
Props: 

-  specialityList (list of speciality)
- userSpeciality (list of selected user speciality)
- maxSelectedItem (max number of speciality which user can select)
- callback (function to return to parent element selected speciality)

### CODE REVIEW

File: src/components/CustomSelect/CustomSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import "./CustomSelect.scss"
+import {arrowSlider} from "../../assets/icons/arrowSlider"
+import { useState } from "react"
+
+export const CustomSelect = (props:{specialityList:Array<string>, userSpeciality:Array<string>, maxSelectedItem:number, callback:Function}) => {
```
Comment (AmidaWolf) [inline src/components/CustomSelect/CustomSelect.tsx:5]: `specialityList:Array<string>` - better, if we use `string[]`, this is more commonly used and generally preferred for its brevity and readability, better with typical TypeScript conventions, but not necessary

File: src/components/CustomSelect/CustomSelect.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        }
+       }
+    }
+  }
+  &-Body {
+    position: absolute;
+    width: 100%;
+    left: 0;
+    top: 100%;
+    border: 1px solid #A9A9A9;
```
Comment (AmidaWolf) [inline src/components/CustomSelect/CustomSelect.scss:60]: border colors or all styles should be used from constants

**Review by AmidaWolf — APPROVED** [review]

File: src/components/CustomSelect/CustomSelect.scss (hunk 2 above)
Comment (Lansyfonseka) [reply src/components/CustomSelect/CustomSelect.scss:60]: I agree. But variables.scss don't have needed constants. I want rework all project structure and create good scss constants

File: src/components/CustomSelect/CustomSelect.tsx (hunk 1 above)
Comment (Lansyfonseka) [reply src/components/CustomSelect/CustomSelect.tsx:5]: Ok, a fix this

**Review by iraskiba — APPROVED** [review]

**Pull request merged**